html/semantics/embedded-content/media-elements/interfaces/TrackEvent/constructor.html

```diff
@@ -13,12 +13,13 @@
     assert_equals(ev.track, null, 'ev.track after assignment');
 }, document.title+', one arg');
 test(function(){
-    var obj = {};
-    var ev = new TrackEvent('foo', {track:obj});
+    var video = document.createElement('video');
+    var testTrack = video.addTextTrack('subtitles', 'foo', 'foo');
+    var ev = new TrackEvent('foo', {track: testTrack});
```

Comment:
Can you add a test for `{track:null}` and `{track:undefined}` and another for a case that should throw TypeError, like `{track:window}` or `{track:{}}`?